lib/php/lib/Transport/TBufferedTransport.php

```diff
@@ -78,11 +69,26 @@ public function __construct($transport = null, $rBufSize = 512, $wBufSize = 512)
      */
     protected $rBuf_ = '';
 
+    /**
+     * Constructor. Creates a buffered transport around an underlying transport
+     */
+    public function __construct($transport, $rBufSize = 512, $wBufSize = 512)
```

Comment:
It would have been more obvious if the pull request changed only this line, removing the "= null".

Reply:
Firstly, remove `=null` at class filed.
And, `$transport=null` should also be removed at function `__construct`.

Sometimes, called `new TBufferedTransport()` will cause $transport_ is null, which cause other funcion doesn't work(`open`, `read`, etc.)